apps/api/src/app/events/usecases/add-job/add-job.usecase.ts

```diff
@@ -22,25 +22,39 @@ export class AddJob {
   ) {}
 
   public async execute(command: AddJobCommand): Promise<void> {
+    Logger.debug('command is: ' + command);
+
+    Logger.verbose('Getting Job');
```

Comment:
Without the proper context such as `environmentId`, orgIf, transactionId those logs seem to me quite useless since we will have millions of them without any context of a request, or am i missing something?